src/sidebar/store/modules/filters.js

```diff
@@ -285,6 +300,7 @@ function hasAppliedFilter(state) {
  * @prop {(filterName: string) => FilterOption|undefined} getFilter
  * @prop {() => Object<string,FilterOption>} getFilters
  * @prop {() => Object<string,string>} getFilterValues
+ * @prop {() => Object<string,FilterOption>} getFocusFilters
```

Comment:
On master it is no longer necessary to write out store module types manually. You'll need to rebase to resolve the conflict.